test/utils/CustomPropTypesSpec.js

```diff
@@ -26,6 +26,30 @@ describe('CustomPropTypes', function() {
     });
   });
 
+  describe('elementType', function () {
+    function validate(prop) {
+      return CustomPropTypes.elementType({p: prop}, 'p', 'Component');
+    }
+
+    function validateRequired(prop) {
+      return CustomPropTypes.elementType.isRequired({p: prop}, 'p', 'Component');
+    }
+
+    it('Should return error with non elementType values', function() {
+      assert.instanceOf(validateRequired(), Error);
+      assert.instanceOf(validateRequired(null), Error);
+      assert.instanceOf(validate(React.createElement('span')), Error);
+      assert.instanceOf(validate({}), Error);
```

Comment:
Testing for existence of an error alone is not enough in many cases. It's usually a good idea to make your assertion against the error message to ensure that you are getting the error you want.